lib/ansible/modules/storage/netapp/na_ontap_service_processor_network.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: na_ontap_service_processor_network
+short_description: Manage NetApp Ontap service processor network
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.6'
+author:
+- Chris Archibald (carchi@netapp.com), Kevin Hutton (khutton@netapp.com)
+description:
+- Modify a Ontap service processor network
+options:
+  state:
+    description:
+    - Whether the specified service processor network should exist or not.
+    choices: ['present']
+    default: present
+  address_type:
+    description:
+    - Specify address class.
+    required: true
+    choices: ['ipv4', 'ipv6']
+  is_enabled:
+    description:
+    - Specify whether to enable or disable the service processor network.
+    required: true
+    choices: ['true', 'false']
+  node:
+    description:
+    - The node where the the service processor network should be enabled
+    required: true
+  dhcp:
+    description:
+    - Specify dhcp type.
+    choices: ['v4', 'none']
+  gateway_ip_address:
+    description:
+    - Specify the gateway ip.
+  ip_address:
+    description:
+    - Specify the service processor ip address.
+  netmask:
+    description:
+    - Specify the service processor netmask.
+  prefix_length:
```

Comment:
`type: int`?

Reply:
@carchi8py it's declared at int below, so this should be a safe change.